pkgs/racket-doc/scribblings/reference/contracts.scrbl

```diff
@@ -1944,8 +1944,8 @@ earlier fields.}}
 
 @deftogether[(@defform[
  #:literals (struct rename)
- (contract-in in-out-item ...)]
-               @defform[
+ (contract-in require-spec in-out-item ...)]
```

Comment:
I don't think `require-spec` is right. It must be a module path. We can't do:

```
(require (contract-in (only-in racket add1)
                      [add1 (-> number? number?)]))
```

which fails with:

```
contract violation
  expected: (and/c syntax? (lambda (s) (module-path? (syntax->datum s))))
  given: #<syntax:3-unsaved-editor:3:22 (only-in racket add1)>
```